### DISCUSSION TITLE
How to track Sleeping time?

### DISCUSSION THREAD

**Discussion by gitraghuram** [super-productivity/super-productivity#3096] [Q&A] [answered]
I would like to track the number of hours I sleep.
But sleep goes past midnight to next day.
So what is the best way to track it using this awesome app?

**Comment by Jagdfalke** [comment, marked as answer]
This is frankly an unusual request for a productivity tool. There is no dedicated way to track sleep time in Super-Productivity.

If you really wanted to track your sleep time, you could create a repeated task named 'Sleeping time' or something similar. And each day when you open the app you could manually enter the time you slept as the time spent on this task. This would however count towards your work time for that day.

**Reply by johannesjo** [reply]
An alternative might be using the simple counter feature.  Should work well imho though you have to enter the stuff manually.

**Comment by gitraghuram** [comment]
At least for some months till i find out where all of my hard-earned time is going.
I am trying to track every minute of the day.
I will try your suggestion of manually entering that information.
